lib/ansible/plugins/filter/core.py

```diff
@@ -103,15 +88,15 @@ def to_nice_json(a, indent=4, *args, **kw):
         else:
             try:
                 major = int(simplejson.__version__.split('.')[0])
-            except:
+            except Exception:
                 pass
             else:
                 if major >= 2:
                     return simplejson.dumps(a, indent=indent, sort_keys=True, *args, **kw)
```

Comment:
This seems like it would create a confusing split, where using `to_nice_json` may or may not represent un-decrypted vault content depending on versions of python / simplejson.

I admit, this will probably never affect me, because I'm not trying to use python 2.6...

Reply:
I have yet to come up with good approach for this, tried using the `default` interface as documented, but that does not seem to work